contracts/contracts/sbtc-registry.clar

```diff
@@ -263,7 +263,15 @@
     ;; Update the current signature threshold
     (var-set current-signature-threshold new-signature-threshold)
     ;; Update the current aggregate pubkey
-    (ok (var-set current-aggregate-pubkey new-aggregate-pubkey))
+    (var-set current-aggregate-pubkey new-aggregate-pubkey)
+    (print {
+      topic: "key-rotation",
+      new-keys: new-keys,
+      new-address: new-address,
+      new-aggregate-pubkey: new-aggregate-pubkey,
+      new-signature-threshold: new-signature-threshold
+    })
+    (ok true)
```

Comment:
nit: I know that `var-set` currently always returns true, but to minimize unintended semantic changes can the `var-set` remain in the return?

Reply:
Eh, would you mind if we kept it as-is? I'd prefer to keep it as (ok true) so that it's the same return message as any other function completed in the 'sbtc-registry' contract (they all end with (ok true)).